crates/bevy_animation/src/lib.rs

```diff
@@ -1234,7 +1234,7 @@ impl Plugin for AnimationPlugin {
                 (
                     advance_transitions,
                     advance_animations,
-                    animate_targets,
+                    animate_targets.after(bevy_render::mesh::morph::inherit_weights),
```

Comment:
I did spot that using https://github.com/jakobhellermann/bevy_mod_debugdump on an unrelated task; glad to see it fixed :tada: 

![image](https://github.com/user-attachments/assets/c9d52698-ce71-4011-b073-3308be562009)